rapid_response_xblock/static/js/src/rapid.js

```diff
@@ -36,6 +36,18 @@
     var timerSpinnerSel = '.timer-spinner';
     var timerSpinnerTextSel = '.timer-spinner-text';
     var numStudentsSel = '.num-students';
+    var tooltipContainerSel = '.rapid-response-tooltip-container';
```

Comment:
I think it's cleaner to create the container element here then `$("body").prepend()` it on page load. That way you don't need to find the element in `renderChart`, you can avoid repeating the class name (as is being done currently when the element is created on page load), and you don't need to check that an element with the same class name exists b/c we are always going to be modifying the object in memory.